std_msgs/Bool request
---
geometry_msgs/Pose robot_curr_pose